sklearn/model_selection/_search.py

```diff
@@ -371,7 +371,7 @@ class BaseSearchCV(six.with_metaclass(ABCMeta, BaseEstimator,
     def __init__(self, estimator, scoring=None,
                  fit_params=None, n_jobs=1, iid=True,
                  refit=True, cv=None, verbose=0, pre_dispatch='2*n_jobs',
-                 error_score='raise'):
+                 error_score='raise', return_train_score=False):
```

Comment:
I know it was in the issue, but I'm not sure why we want to make that optional / not the default. Computing the training scores is really cheap compared to everything else that goes on. There are already a lot of parameters, not sure if this one is particularly helpful. @jnothman ?